src/mmw/apps/modeling/tasks.py

```diff
@@ -265,6 +265,9 @@ def run_srat(watersheds):
         result = r.json()
     except ValueError:
         logger.error('SRAT Catchment API did not return JSON')
+    
+    if (r.status_code != 200):
+        raise Exception('SRAT Catchment API request failed: %s' % result)
```

Comment:
Adding these lines makes SRAT Celery task never finish. Commenting these out makes it work again. Would recommend just logging it for now.